### PR TITLE
 Fix #27 Data too long for column 'desc' 

### LINKED ISSUE #27: Implement validator for TaskModel.desc field or crop to 64 char?
Hi,
I had a strange error `error - (1406, "Data too long for column 'desc' at row 1")`
After browsing the net, and then exploring the code of huey-monitor,

I finally manage to realize the reason my task was failling was that the description I was setting for my `ProcessInfo `was 67 char while only 64 were authorized by the `TaskModel `model.

This error was quite small, and may happen easily ... but it was quite hard to find-out.

I therefore would suggest to make this error much more easy to find-out for users.

One solution could be to implement a custom validator for the `TaskModel.desc` field to document specifically the error for the user.

However, having a `desc `bigger than 64 char should not for me prevent a task to run. 
I would therefore suggest another simpler solution: 
to implement in the `ProcessInfo ` model a crop to 64 char when updating `TaskModel`.

A minimal implementation could be:

```
TaskModel.objects.filter(task_id=task.id).update(
            desc=self.desc[:64],
            total=self.total,
            unit=self.unit,
            unit_divisor=self.unit_divisor,
        )
```

### PR DESCRIPTION
**Pull request by jedie**

### CODE REVIEW

File: huey_monitor/tqdm.py [hunk 1]
```diff
@@ -40,6 +41,15 @@ def __init__(self,
         self.unit_divisor = unit_divisor
         self.parent_task_id = parent_task_id
 
+        if len(self.desc) > 64:
+            # We call .update() that will not validate the data, so a overlong
+            # desciption will raise a database error and maybe a user doesn't know
```
Comment (phihag) [inline huey_monitor/tqdm.py:46]: typo should be `description`

File: huey_monitor/tqdm.py [hunk 2]
```diff
@@ -40,6 +41,15 @@ def __init__(self,
         self.unit_divisor = unit_divisor
         self.parent_task_id = parent_task_id
 
+        if len(self.desc) > 64:
```
Comment (phihag) [inline huey_monitor/tqdm.py:44]: Can we dynamically determine the length of the description field?

Comment (jedie) [reply huey_monitor/tqdm.py:44]: Yes, i do this in the test:

`max_length = TaskModel._meta.get_field('desc').max_length`

but i think it's faster to use a static value...
If this value is not the right one, the test will fail.

File: huey_monitor/tqdm.py (hunk 1 above)
Comment (jedie) [reply huey_monitor/tqdm.py]: thanks. fixed.

**Pull request merged**